test/integration/t8n/CMakeLists.txt

```diff
@@ -57,3 +57,38 @@ set_tests_properties(
     FIXTURES_CLEANUP ${TEST_CASE}
     PASS_REGULAR_EXPRESSION ${EXPECTED_OUT_ALLOC}
 )
+
+set(TEST_CASE osaka_empty_requests)
```

Comment:
I'm fine with it, but requests are specific to Prague

Reply:
ah, of course needs fixing :facepalm: thx